homeassistant/components/edl21/sensor.py

```diff
@@ -397,7 +398,7 @@ async def add_entities(self, new_entities: list[EDL21Entity]) -> None:
             old_entity_id = registry.async_get_entity_id(
                 "sensor", DOMAIN, entity.old_unique_id
             )
-            if old_entity_id is not None:
+            if old_entity_id is not None and entity.unique_id is not None:
```

Comment:
This migration code has been in place for almost 3 years: #33594
Maybe it is time to remove the migration path and the `old_unique_id` property (in a preliminary or follow-up PR)?